src/components/spreadsheet/utils/config-tables.js

```diff
@@ -56,6 +61,35 @@ const propertiesGetter = (params) => {
         return null;
     }
 };
+const handleCellClickEvent = (event) => {
```

Comment:
it better to give another name to this function, because we may register more callback for other cells.